R/zzz.R

```diff
@@ -320,16 +323,9 @@ settings <- new.env(parent = emptyenv())
     exclude_linter_sep = rex(any_spaces, ",", any_spaces),
     exclusions = list(),
     cache_directory = R_user_dir("lintr", "cache"),
-    comment_token = Sys.getenv("GITHUB_TOKEN", unset = NA) %||% rot(
-      paste0(
-        "0n12nn72507",
-        "r6273qnnp34",
-        "43qno7q42n1",
-        "n71nn28"
-      ),
-      54L - 13L
-    ),
-    error_on_lint = logical_env("LINTR_ERROR_ON_LINT") %||% FALSE
+    # rot("0n12nn72507r6273qnnp3443qno7q42n1n71nn28", 54L - 13L) [rot from ?chartr]
+    comment_token = Sys.getenv("GITHUB_TOKEN", unset = "0a12aa72507e6273daac3443dab7d42a1a71aa28"),
```

Comment:
Pretty sure we can just remove this altogether?

https://github.com/search?q=repo%3Ar-lib%2Flintr%20%2Fcomment_token%2F&type=code

Reply:
True. I think it's a remnant of the old github integration via comments.
Let's remove it (maybe in a separate PR to avoid scope creep?)